src/app/demo/vue/vue.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { VueComponent } from './vue.component';
+
+// TODO: Fix this test suite.
```

Comment:
We are getting a lot of these. Is there an open issue for this? I'd be happy to contribute to unblock new tests with some helpers or something. It seems like we skip this because it is too difficult to mock out some very common directives/components.